video/out/wayland_common.c

```diff
@@ -2529,21 +2533,22 @@ static void set_color_management(struct vo_wayland_state *wl)
         return;
     struct pl_hdr_metadata hdr = wl->target_params.color.hdr;
     if (wl->supports_display_primaries) {
-        xx_image_description_creator_params_v4_set_mastering_display_primaries(wl->image_creator_params,
+        wp_image_description_creator_params_v1_set_mastering_display_primaries(wl->image_creator_params,
                 hdr.prim.red.x * WAYLAND_COLOR_FACTOR, hdr.prim.red.y * WAYLAND_COLOR_FACTOR, hdr.prim.green.x * WAYLAND_COLOR_FACTOR,
                 hdr.prim.green.y * WAYLAND_COLOR_FACTOR, hdr.prim.blue.x * WAYLAND_COLOR_FACTOR, hdr.prim.blue.y * WAYLAND_COLOR_FACTOR,
                 hdr.prim.white.x * WAYLAND_COLOR_FACTOR, hdr.prim.white.y * WAYLAND_COLOR_FACTOR);
 
         // No clear enum in the protocol to indicate if these are supposed
         // to work or not so just hope they do if setting display
         // primaries is supported.
```

Comment:
Sorry one last thing I noticed. This comment can be removed. It's pretty obvious that all of these are a part of mastering display primaries now.

Reply:
done